tests/SampleApps/DotNetCore/NetCore7PreviewMvcApp/NetCore7PreviewMvcApp.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Change to the `TargetFramework` to point at the .NET Core 7 Preview framework

Reply:
Updated. Thanks!